README.md

```diff
@@ -328,6 +328,13 @@ It uses a number of command line options to specify how to display job informati
 * `-w, --why`: show why a job was held
 * `-m, --matched`: show site and machine to which the job matched (for CMS Connect)
 * `-p, --progress`: show job progress (time and nevents)
+* `-X, --xrootd-resubmit`: resubmit the jobs based on where the input files are located
+* `-C INPUTFILECLASSAD, --input-file-classad=INPUTFILECLASSAD`: HTCondor ClassAd which contains the input file(s) being used within the job (used in combination with `-X`)
+* `-D, --dry-run`: don't actually resubmit any jobs (used in combination with `-X`)
+* `-K LOGKEY, --log_key=LOGKEY`: key to use to find the correct line(s) in the log file (used in combination with `-X`)
+* `-L LOGPATH, --log_path=LOGPATH`: path to the job logs (used in combination with `-X`, default = `pwd`)
+* `-U, --prefer-us-sites`: preferentially select US sites over others (used in combination with `-X`)
```

Comment:
here, we might want to be more specific: "prefer reading inputs from US sites over others"
(since there are also arguments to specify the site where the job runs, which is different)